tests/countries/test_seychelles.py

```diff
@@ -0,0 +1,181 @@
+#  python-holidays
+#  ---------------
+#  A fast, efficient Python library for generating country, province and state
+#  specific sets of holidays on the fly. It aims to make determining whether a
+#  specific date is a holiday as fast and flexible as possible.
+#
+#  Authors: dr-prodigy <dr.prodigy.github@gmail.com> (c) 2017-2023
+#           ryanss <ryanssdev@icloud.com> (c) 2014-2017
+#  Website: https://github.com/dr-prodigy/python-holidays
+#  License: MIT (see LICENSE file)
+
+from unittest import TestCase
+
+from holidays.countries.seychelles import Seychelles, SC, SYC
+from tests.common import CommonCountryTests
+
+
+class TestSeychelles(CommonCountryTests, TestCase):
+    @classmethod
+    def setUpClass(cls):
+        years = range(1994, 2050)
+        super().setUpClass(Seychelles, years=years, years_non_observed=years)
```

Comment:
```suggestion
        super().setUpClass(Seychelles)
```
Since we don't have tests for entire range of years.